internal/v1/server.go

```diff
@@ -135,6 +151,49 @@ func Attach(conf *ServerConfig) error {
 	})
 
 	h.server.echo.GET("/metrics", echo.WrapHandler(promhttp.Handler()))
+
+	var nonFoundRoutes []string
+
+	// as we need to implement some path registrations manually, let's check if all are there
+	mapOpenAPISyntax := getMapOpenAPISyntax()
```

Comment:
this chuck sounds like a dev helper, but nor something we need in prod, I'm not sure if we need it and if it's still just to have the `/openapi.json` route in a spec, my vote goes for getting rid of it from there.

It doesn't add any value to have it there. You need to know how to retrieve the spec to get any info from it and once you have the spec, you do not need it to contain info about how to retrieve spec - because you already have it! :)